src/anshuKJun20/AppendLastString.java

```diff
@@ -0,0 +1,24 @@
+package anshuKJun20;
+
+public class AppendLastString {
+	
+	void appendLastString(String input ,String requiredOutput) {
+		int count =0;
+		if(input.length() == requiredOutput.length()) {
+			while(!input.equals(requiredOutput)) {
```

Comment:
what if String can't be made equals even after all possible rotation ?? this will run infinitely ...

you can handled it with one more condition.
while(!input.equals(requiredOutput) && count< input.length)